Sources/TagCell.swift

```diff
@@ -8,21 +8,35 @@ protocol TagCellDelegate: class {
 class TagCell: UICollectionViewCell {
     static let identifier = "TagCell"
 
+    static func widthFor(tagTitle title: String) -> CGFloat {
```

Comment:
This method is called from the `sizeForItemAt` delegate method from the CollectionViewLayout. So it gives the tag width before the tagCell is actually loaded.